target/linux/bcm27xx/image/distroconfig.txt

```diff
@@ -12,5 +12,7 @@ dtoverlay=disable-bt
 dtoverlay=disable-bt
 [pi4]
 dtoverlay=disable-bt
+[pi5]
+dtoverlay=disable-bt
```

Comment:
The Raspberry Pi 5 doesn't have a mini UART, so is this needed? Or was it copied from pi4 without checking?
https://www.raspberrypi.com/documentation/computers/configuration.html#raspberry-pi-5

Reply:
rpi5 bluetooth work with https://github.com/openwrt/openwrt/pull/10341
```
[   15.939765] Bluetooth: hci0: BCM: features 0x2f
[   15.945700] Bluetooth: hci0: BCM4345C0 Ampak 0073
[   15.950418] Bluetooth: hci0: BCM4345C0 (003.001.025) build 0075
[   15.956856] Bluetooth: hci0: BCM: Using default device address (44:45:c0:00:22:ac)
```